CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Removed Terra prefix from mixin pattern rule.
```

Comment:
This can be moved under the other "Changed" section.

Reply:
Resolved in 968992f